Source/Core/Core/PowerPC/Jit64/Jit.h

```diff
@@ -36,6 +36,11 @@ struct CodeOp;
 
 class Jit64 : public Jitx86Base
 {
+  // these three need access to the JitState
```

Comment:
This is self-evident by the use of the friend declarations

Reply:
I'm pretty sure this is part of #7602 …
Anyway, I put this comment there to make clear that it is JitState (and not some other member) that is accessed by these classes.